CHANGELOG.md

```diff
@@ -2,6 +2,59 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
+2.25.0 (in progress)
+====================
+
+New Features
+------------
+
+**Auto-switch default audio input devices**
+
```

Comment:
```suggestion

This release adds a new property that handles the device management required to preserve audio continuity in situations where end-users change the default audio input device. 

```

Reply:
Also is there a github issue associated with this? Would be good to link to it if so.

Reply:
No. This was reported by a customer in an escalation.

Reply:
It sounds like this could address https://github.com/twilio/twilio-video-app-react/issues/540